ush/setup.py

```diff
@@ -381,22 +411,25 @@ def get_location(xcs,fmt):
     try:
         UFS_WTHR_MDL_DIR = get_ini_value(cfg, external_name, property_name)
     except KeyError:
-        errmsg = dedent(f'''
-                        Externals configuration file {mng_extrns_cfg_fn}
-                        does not contain "{external_name}".''')
+        errmsg = dedent(
+            f"""
+            Externals configuration file {mng_extrns_cfg_fn}
+            does not contain "{external_name}"."""
```

Comment:
Should the quotes around `{external_name}` be double or single?

Reply:
I will replace all instances like this to use double quotes outside and single quotes within.